backend/app/models/user.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'bcrypt'
+require 'securerandom'
+
+# Create user model
+class User < ApplicationRecord
+  attr_reader :password
+
+  validates :username, presence: true, uniqueness: true
+  validates :email, presence: true, uniqueness: true
+  validates :password_digest, :session_token, presence: true
```

Comment:
Uh oh. Session token should be unique too. If it weren’t, I could hijack who the current user is if I can manage to get a two users to have the same session token

Reply:
Okay, will change session token to be unique.